tests/cypress/integration/actions_users/issue_1810_login_logout.js

```diff
@@ -10,17 +10,19 @@ context('When clicking on the Logout button, get the user session closed.', () =
     const issueId = '1810';
     let taskId;
 
+    function closeModal() {
+        if (Cypress.browser.family !== 'chromium') {
```

Comment:
Can we put the condition withing closeModalUnsupportedPlatform?

Reply:
Yes. Done.